### DISCUSSION TITLE
Enable a specific profile based on account launched

### DISCUSSION THREAD

**Discussion by jjccharles** [runelite/runelite#16307] [Ideas]
Ahoy,

Profiles are turning out to be incredibly useful for reducing the amount of time spent turning on/off certain plugin settings when moving to a different context.

I am finding that the profiles system is really useful on a per account basis, so I was wondering if there would be a way to add a default profile based on the account being logged into. 

Jagex launcher use case:
1. Player launches runelite via Jagex launcher
2. Runelite identifies its account x, loads profile x
3. Runelite at login screen with correct profile loaded

Standard use case:
1. Player launches runelite, it boots into the first profile
2. Player logs in with account x, loads profile x on login

To be fair can see how the latter could be problematic if you add in extra loading when the player is gaining control. Even for the first use case, would it be possible as a plugin hub plugin?

**Comment by Nightfirecat** [comment]
I'm hoping to add this in the coming couple of weeks.
